bin/srcflux

```diff
@@ -247,21 +247,18 @@ def validate_src_and_bkg_regions( myroot, infile, tmpdir, src, bkg ):
         dmmakereg.verbose=0
         dmmakereg()
 
-        rr=regParse( "region({})".format(dmmakereg.outfile ))
+        rr=CXCRegion(dmmakereg.outfile)
         gorm( tfile )
-        return regRegionString( rr )
+        return str(rr)
 
 
     def validate_region( reg ):
+        "See if it is a file, need to open w/ DM to check filters/etc"
         try:
-            rr = regParse( reg )
+            read_file(reg)
+            reg = f"region({reg})"
         except:
-            reg2 = "region({})".format(reg)
-            try:
-                rr = regParse( reg2 )
-                reg=reg2
-            except:
-                raise ValueError("Could not parse '{}' as either a region string or a region file".format(reg))
+            pass
```

Comment:
Looking at this diff it looks like we've lost the ability to check if region is not a valid region string. Now, this is probably checked for in `regino_to_phys`, but if I'm reading the code correctly you'll just get an error frmo `dmmakereg` rather than the previous, probably-easier-to-understand error.

Or is this case now checked earlier?

Reply:
Good point.  You would have gotten something from `dmmakereg` but probably better to catch here.

I've updated the `validate` routine to add additional checks in e5fd4e7